src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs

```diff
@@ -2702,6 +2185,231 @@ private static IntPtr GetAddressOfVariantAtIndex(IntPtr variantArrayPtr, uint in
                 return (IntPtr)((ulong)variantArrayPtr + ((ulong)index) * ((ulong)variantSize));
             }
         }
+
+        private class SystemIAccessibleDecorator : IAccessible
```

Comment:
We call such classes wrappers.
```suggestion
        private class SystemIAccessibleSafeWrapper : IAccessible
```

Also please add xml-doc explaining the purpose of the class

Reply:
Added the xml-doc, thanks.

Reply:
Renamed class.